vizro-core/src/vizro/static/css/table.css

```diff
@@ -0,0 +1,56 @@
+div.right_side
+  .dash-table-container
+  .dash-spreadsheet-container
+  .dash-spreadsheet-inner
+  th {
+  background-color: var(--main-container-bg-color);
+}
+
+div.right_side
+  .dash-table-container
+  .dash-spreadsheet-container
+  .dash-spreadsheet-inner
+  td {
+  background-color: var(--main-container-bg-color);
+}
+
+div.right_side
+  .dash-table-container
+  .dash-spreadsheet-container
+  .dash-spreadsheet-inner
+  table {
+  --hover: var(--main-container-bg-color);
+}
+
+div.right_side
+  .dash-table-container
+  .dash-spreadsheet-container
+  .dash-spreadsheet-inner
+  td
+  .dash-cell-value.unfocused {
+  color: var(--text-primary);
+  font-size: 14px;
+}
+
+div.right_side
+  .dash-table-container
+  .dash-spreadsheet-container
+  .dash-spreadsheet-inner
+  .column-header-name {
+  color: var(
+    --text-primary
+  ); /* Should be text-secondary when hover color is implemented */
+  font-size: 14px;
+}
+
+.table-title {
+  padding-bottom: 14px;
+}
+
+.table-container {
+  height: 100%;
+  width: 100%;
+  overflow: auto;
+  padding: 12px;
+  padding-left: 20px;
+}
```

Comment:
Sorry, I know this is a bit tedious. Seems like we don't need the title `padding` after all (the gap between the graph title and the graph seem to be of the same gap as the table title and the table now). I also think we need to increase the `padding-left`, such that the table does align with the y-axis title:

![Screenshot 2023-10-25 at 21 05 50](https://github.com/mckinsey/vizro/assets/90609403/d3636100-2f89-42af-b5f9-7aa5bda9af75)

I kind of don't like that the graph and table title don't align, but I think this is our initial design on the charts. I'll create an issue for that such that we re-evaluate the padding of the graph title, but feel free to ignore this one right now.

```suggestion
.table-container {
  height: 100%;
  width: 100%;
  overflow: auto;
  padding: 12px;
  padding-left: 32px;
}
```